gym_bridges/renderer/test_pygame_renderer.py

```diff
@@ -0,0 +1,19 @@
+import numpy as np
+import pytest
+import time
+
+from gym_bridges.envs.bridges_env import BridgesEnv
+
+
+@pytest.mark.parametrize("width", [3, 9])
+def test_force_standard_config_render(width):
+    """The standard configuration only has ground at the bottom left
+    and right locations of the env."""
+    env = BridgesEnv(width=width, force_standard_config=True)
+    height = env.shape[0] - 1
+    initial_state = env.reset()
+    env.render(mode="pygame")
+    assert initial_state[height, 0] == BridgesEnv.StateType.GROUND
```

Comment:
I appreciate the desire to test very much!

I'm not sure this test does much though and I'm not sure how we'd test the pygame mechanically. I'd be ok with this being untested for now. 

One crazy idea could be if pygame lets one "save" the screen and then comparing to a golden file that it 'looks right'. I'm not convinced this effort (or approach) is necessary right now.